src/main/java/de/tum/in/www1/artemis/service/util/VCSSimulationUtils.java

```diff
@@ -0,0 +1,23 @@
+package de.tum.in.www1.artemis.service.util;
+
+import java.security.SecureRandom;
+
+import org.springframework.context.annotation.Profile;
+
+@Profile("dev")
+public class VCSSimulationUtils {
+
+    /**
+     * Simulates a commit Hash, the returned string consist out of 40 times the same number
+     * @return the simulated commitHash
+     */
+    public static String simulateCommitHash() {
+        SecureRandom secureRandom = new SecureRandom();
+        String number = String.valueOf(secureRandom.nextInt(10));
+        String commitHash = number;
```

Comment:
This can be changed in the follow-up PR:
You can simplify this code (and make it more readable in my opinion) by using the `Strings` class from `com.google.common.base.Strings`.

The code then looks like this:
```
return Strings.repeat(number, 40);
```